watcher/tool/watcher/alerts.go

```diff
@@ -115,7 +115,7 @@ func receiverLoop(ctx context.Context, kubeClient *kubeapi.Clientset, kClient *k
 	}
 }
 
-func createAlert(client *kapacitor.Client, spec []byte, log *log.Entry) error {
+func createAlert(client *kapacitor.Client, spec []byte, log *logrus.Entry) error {
```

Comment:
We usually end up passing `logrus.FieldLogger` interface instead of concrete `logrus.Entry` around as this allows stubbing logging with custom implementation if necessary.